app/views/account/admin/users/_user.html.slim

```diff
@@ -2,15 +2,9 @@ tr
   td= user.email
   td= user.full_name
   td
-    - if admin_or_moderator? && user.reporter?
-      - if user.banned?
-        = link_to 'Розблокувати', toggle_ban_account_admin_user_path(user), method: :patch
-      - else
-        = link_to 'Заблокувати', toggle_ban_account_admin_user_path(user), method: :patch
+    = link_to bann_user?(user), bann_path(user), method: :patch
```

Comment:
What will be rendered on page if user is an admin? 
We shouldn't render this element if helpers return nil or empty string. So you can write a helper which will return whole element when it needed.